src/render/direct3d11/SDL_render_d3d11.c

```diff
@@ -1615,12 +1615,11 @@ D3D11_QueueDrawPoints(SDL_Renderer * renderer, SDL_RenderCommand *cmd, const SDL
 {
     VertexPositionColor *verts = (VertexPositionColor *) SDL_AllocateRenderVertices(renderer, count * sizeof (VertexPositionColor), 0, &cmd->data.draw.first);
     int i;
-    const SDL_Color color = {
-        .r = cmd->data.draw.r,
-        .g = cmd->data.draw.g,
-        .b = cmd->data.draw.b,
-        .a = cmd->data.draw.a
-    };
+    SDL_Color color;
+    color.r = cmd->data.draw.r;
+    color.g = cmd->data.draw.g;
+    color.b = cmd->data.draw.b;
+    color.a = cmd->data.draw.a;
 
```

Comment:
You can just do: color = cmd->data.draw

Reply:
Unfortunately the `draw` structure isn't an `SDL_Color`. I was thinking of changing that but maybe for another PR.